src/vue/pages/SaleDetails.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="sale-details">
+    <top-bar v-if="sale.id">
```

Comment:
v-if=sale and make sale null by default

Reply:
we're interested exactly in the existence of the sale record, not it's ID

Reply:
Fixed